md/docs/derby-0.6/models/backends.md

```diff
@@ -44,7 +49,7 @@ Racer paths are translated into database collections and documents using a natur
 collection.documentId.documentProperty
 ```
 
-LiveDB Mongo will add `_type`, and `_v` properties to Mongo documents for internal use. It will strip out these properties as well as `_id` when it returns the document from Mongo. If a document is an object, it will be stored as the Mongo document directly. If it is another type, it will be nested under a property on the Mongo document caleld `_data`. For object documents, Racer will set the `id` property on the document object in the model automatically.
+ShareDB Mongo will add `_type`, and `_v` properties to Mongo documents for internal use. It will strip out these properties as well as `_id` when it returns the document from Mongo. If a document is an object, it will be stored as the Mongo document directly. If it is another type, it will be nested under a property on the Mongo document called `_data`. For object documents, Racer will set the `id` property on the document object in the model automatically.
```

Comment:
also adds `_m`. I'd note this, document the default properties (ctime and mtime), and document that additional metadata properties can be added with middleware that modifies `snapshot.m` in apply or commit.